modules/ocl/cl_image_warp_handler.cpp

```diff
@@ -156,91 +177,14 @@ CLImageWarpKernel::post_execute (SmartPtr<DrmBoBuffer> &output)
         _warp_frame_id ++;
         XCAM_LOG_DEBUG ("@DEBUG POP Image channel(%d), input frame id(%d)", _channel, _input_frame_id);
         XCAM_LOG_DEBUG ("@DEBUG Warp config id(%d), Warp image id(%d)", _warp_config.frame_id, _warp_frame_id);
-        XCAM_LOG_DEBUG ("@DEBUG image list size(%d)", _image_in_list.size());
+        XCAM_LOG_DEBUG ("@DEBUG image list size(%lu)", _image_in_list.size());
```

Comment:
remove all "@DEBUG" in XCAM_LOG_DEBUG,  the macro definition already have DEBUG header.